comps/third_parties/sglang/src/Dockerfile

```diff
@@ -30,7 +30,7 @@ RUN curl -fsSL -v -o miniforge.sh -O https://github.com/conda-forge/miniforge/re
 RUN git clone https://github.com/jianan-gu/sglang -b llama4_optimzed_cpu_r1
 RUN . ~/miniforge3/bin/activate && conda create -n sglang python=3.10 && conda activate sglang && \
     cd sglang && pip install -e "python[all_cpu]" && cd .. && conda install -y libsqlite=3.48.0 && \
-    pip uninstall -y triton && pip uninstall -y transformers && pip install transformers==4.51.1 && \
+    pip uninstall -y triton && pip uninstall -y transformers && pip install transformers>=4.50.0 && \
```

Comment:
This case is not actually being tested in CI, https://github.com/opea-project/GenAIComps/blob/main/tests/third_parties/_test_third_parties_sglang.sh, underscore before test scripts means not ready for test.

Reply:
It's good to know.
We can patch it if it fails during manual testing.
Thanks for raising this concern.